include/experimental/__p0009_bits/layout_left.hpp

```diff
@@ -176,6 +176,12 @@ class layout_left::mapping {
     MDSPAN_INLINE_FUNCTION constexpr bool is_exhaustive() const noexcept { return true; }
     MDSPAN_INLINE_FUNCTION constexpr bool is_strided() const noexcept { return true; }
 
+    MDSPAN_TEMPLATE_REQUIRES(
```

Comment:
I don't like introducing a template parameter here for C++17/14. I would say we just do this only for C++20 using a requires clause on a non-templated function, and note down in the readme that this constraint is not implemented pre-c++20,